skeleton/docker-entrypoint.sh

```diff
@@ -97,7 +97,7 @@ fi
 $sudo $VENVBIN/python /app/scripts/cors.py
 
 # Handle ADDONS installation
-if [[ -v ADDONS ]]; then
+if [[ -z "${ADDONS}" ]]; then
```

Comment:
@sneridagh I think you want `-n`. `-z` is true if the string is empty, so then we would only try to install the addons if there are none.

Reply:
Uh, ok! Changing! I was misguided by another post then... my bash-fu is not good :P

Reply:
@davisagli Tried it out and works well!!